### DISCUSSION TITLE
Other Steam Review Editors?

### DISCUSSION THREAD

**Discussion by dainank** [dons20/steam-review-editor#12] [Q&A] [answered]
First off, this is great work and works flawlessly for me!

I had trouble finding anything else doing similar stuff, thus I was wondering, during your endeavor of developing this, **did you stumble across any other _Steam_ review editors online?** It seems rather barren even though _Steam_ is a huge platform with thousands of people writing reviews daily.

**Comment by dons20** [comment, marked as answer]
Thank you, I'm happy to hear that the tool worked well for you 😄 

Surprisingly no. I created this tool during a time when I was writing a lot more reviews and I wanted to have a more seamless experience while editing. The built-in editor on steam was okay enough, but it didn't offer any previews, so I would usually have to save my review as friends only first to ensure it displayed properly, and then I'd make it public afterwards.

I searched around for a tool that was built for this purpose, but at that time, I found zero. It does seem like my tool is still one of the only tools that fills this niche. My only regret is not having enough time to finish the upgrade / rework sooner. But I aim to finish it soon though.
